dali/python/nvidia/dali/experimental/dynamic/_tensor.py

```diff
@@ -12,16 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import copy
+import warnings
 from typing import Any, Optional, Tuple, Union
-from ._type import DType, dtype as _dtype, type_id as _type_id
-from ._device import Device, device as _device
+
 import nvidia.dali.backend as _backend
-from ._eval_context import EvalContext as _EvalContext
-from . import _eval_mode
-from . import _invocation
-import copy
 import nvidia.dali.types
-import warnings
+
+from . import _eval_mode, _invocation
+from ._device import Device
+from ._device import device as _device
+from ._eval_context import EvalContext as _EvalContext
+from ._type import DType
+from ._type import dtype as _dtype
+from ._type import type_id as _type_id
```

Comment:
Also an option I think:
```suggestion
from . import _eval_mode, _invocation
from ._device import Device, device as _device
from ._eval_context import EvalContext as _EvalContext
from ._type import DType, dtype as _dtype, type_id as _type_id
```